plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -1017,8 +780,146 @@ class GesturePluginTest {
   fun obtainMotionEventActionLater(action: Int): MotionEvent {
     return MotionEvent.obtain(200, 500, action, 15.0f, 10.0f, 0)
   }
+}
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+class FlingGestureTest(
+  private val targetPitch: Double,
+  private val targetScrollMode: ScrollMode,
+  private val velocity: Pair<Float, Float>,
+  private val expectedCoordinate: ScreenCoordinate
+) {
+  private val context: Context = mockk(relaxed = true)
+  private val attrs: AttributeSet = mockk(relaxUnitFun = true)
+  private val mapDelegateProvider: MapDelegateProvider = mockk(relaxUnitFun = true)
+  private val gesturesManager: AndroidGesturesManager = mockk(relaxed = true)
+  private val mapCameraManagerDelegate: MapCameraManagerDelegate = mockk(relaxUnitFun = true)
+  private val mapTransformDelegate: MapTransformDelegate = mockk(relaxUnitFun = true)
+  private val mapPluginProviderDelegate: MapPluginProviderDelegate = mockk(relaxUnitFun = true)
+  private val cameraAnimationsPlugin: CameraAnimationsPlugin = mockk(relaxed = true)
+
+  private lateinit var presenter: GesturesPluginImpl
+
+  private val typedArray: TypedArray = mockk(relaxed = true)
+  private val pack = "com.mapbox.maps"
+
+  @Before
+  fun prepare() {
+    mockkObject(GesturesAttributeParser)
+    every {
+      GesturesAttributeParser.parseGesturesSettings(
+        context,
+        attrs,
+        any()
+      )
+    } returns GesturesSettings()
+
+    every { context.obtainStyledAttributes(any(), any(), 0, 0) } returns typedArray
+    every { context.packageName } returns pack
+    every { typedArray.getString(any()) } returns "pk.token"
+    every { typedArray.getBoolean(any(), any()) } returns true
+    every { typedArray.getDimension(any(), any()) } returns 10.0f
```

Comment:
do you really need all this `every` calls? it's slow down every test. better to remove useless

Reply:
The test execution covers these code paths, otherwise the test will fail with unmockked methods.